src/main/scala/io/viash/platforms/NextflowPlatform.scala

```diff
@@ -180,8 +180,10 @@ case class NextflowPlatform(
     val processLabels = config.labels.map{ case (k, v) => s"withLabel: $k { $v }"}
     val inlineScript = config.script.toList
 
+    val name = f"${functionality.namespace.map(_ + "/").getOrElse("")}${functionality.name}"
```

Comment:
Do we need to adjust this to namespace_sep?

Reply:
No -- the namespace sep is only used for determining the docker container id